tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -774,6 +774,50 @@ var _ = ginkgo.Describe("[tidb-operator] TiDBCluster", func() {
 		framework.ExpectEqual(existed, true)
 		framework.ExpectEqual(value, label.TiDBOperator)
 	})
+
+	ginkgo.It("[Feature: AdvancedStatefulSet] Upgrading tidb cluster while pods are not consecutive", func() {
+		if !ocfg.Enabled(features.AdvancedStatefulSet) {
+			framework.Skipf("AdvancedStatefulSet feature of default operator is not enabled, skipping")
+		}
```

Comment:
right now we have a dedicated AdvnacedStatefulSet job, so we can run it in parallel. of cause, we should skip it in non-AdvnacedStatefulSet job.